sdk/spring/scripts/compatibility_update_supported_version_matrix_json.py

```diff
@@ -32,6 +32,13 @@ def get_args():
         help='Which major version of Spring Boot to use. The default is ' + SPRING_BOOT_MAJOR_3_VERSION_NAME + '.'
     )
     parser.add_argument('-mcp', '--matrix-config-path', type=str, default='sdk/spring/pipeline/supported-version-matrix.json')
+    parser.add_argument(
+        '--use-the-latest-version',
+        type=str,
```

Comment:
use boolean instead

Reply:
If use boolean, then the default value is False, I expect the default value is null to combability with live test usage.